Geometry/HcalTowerAlgo/test/HcalGeometryPlan1Tester.cc

```diff
@@ -57,16 +57,17 @@ void HcalGeometryPlan1Tester::analyze(const edm::Event& /*iEvent*/,
   std::cout << "Special RBX Flag " << ok << " with " << idsp.size()
 	    << " ID's" << std::endl;
   int nall(0), ngood(0);
-  for (unsigned int k=0; k<idsp.size(); ++k) {
-    if (topology.valid(idsp[k])) {
+  for (std::vector<HcalDetId>::const_iterator itr=idsp.begin();
```

Comment:
@bsunanda - the following would be shorter. Please, have a look at http://en.cppreference.com/w/cpp/language/range-for
```
for( const auto& itr : idsp )
{
 ...
} 
```